src/main/build.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+This file is part of the DITA Open Toolkit project.
+
+Copyright 2006 IBM Corporation
+
+See the accompanying LICENSE file for applicable license.
+-->
+<project name="DOST" default="init">
+
+  <!-- Initialize DITA-OT base directory -->
+  <dirname property="ant.file.DOST.dir" file="${ant.file.DOST}" />
+  <property name="dita.dir" location="${ant.file.DOST.dir}"/>
+  <!-- XXX Is this ever used? -->
```

Comment:
I see `dita.dir` used in tons of plugins and our generated `build.xml` uses it quite a bit ... but I'm guessing this is a question about the second definition? I assume that second definition wouldn't be used...